src/components/inputs/button/button.component.tsx

```diff
@@ -162,6 +191,9 @@ export const Button: FC<ButtonProps> = ({
       startIcon={icon}
       type={type}
       variant={variant}
+      component={component}
+      accept={accept}
```

Comment:
That will not suffice Ana. The user passes an array to the prop because you asked him/her to, that's fine. However, MUI's button prop expects a string. So, you have to spread the array, right?